examples/js/getting-started/src/app.js

```diff
@@ -1,25 +1,26 @@
 import { liteClient as algoliasearch } from 'algoliasearch/lite';
 import instantsearch from 'instantsearch.js';
-import { carousel } from 'instantsearch.js/es/templates';
 import {
   configure,
   hits,
   pagination,
   panel,
   refinementList,
   searchBox,
-  trendingItems,
 } from 'instantsearch.js/es/widgets';
 
 import 'instantsearch.css/themes/satellite.css';
 
 const searchClient = algoliasearch(
-  'latency',
-  '6be0576ff61c053d5f9a3225e2a90f76'
+  'DIYPADIATS',
+  'c96176e1b36590680fb3d36bc480d592',
+  { authMode: 'WithinHeaders' }
```

Comment:
wonder if this breaks something. Ideally composition API supports withinQueryParams so that it's a SimpleRequest